File: programming files/even_odd_site.py
# Opens file numbers.html with the ability to write to the file
with open("numbers.html", "w") as f:
    # Writes html code to the file
    f.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n")
    f.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n")
    # Adds table row elements 50 times
    for i in range(1, 50):
        if i % 2 == 0:
            f.write("<tr><td>{}</td><td></td></tr>\n".format(i))
        else:
            f.write("<tr><td></td><td>{}</td></tr>\n".format(i))
    f.write("</table>\n</body>\n</html>")
# Reads the file
with open("numbers.html", "r") as f:
    print(f.read())
    

File: programming files/even_odd_site.java
// imports package
import java.io.*;

public class even_odd_site {
    public static void main(String[] args) throws IOException {
        // Creates object to write to html file
        BufferedWriter file = new BufferedWriter(new FileWriter("numbers.html"));
        // Adds html code to file
        file.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n");
        file.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n");
        // Loops 50 times to add table row headers to file
        for (int i = 1; i <= 50; i++) {
            if (i % 2 == 0) {
                file.write("<tr><td>" + i + "</td><td></td></tr>\n");
            }
            else {
                file.write("<tr><td></td><td>" + i + "</td></tr>\n");
            }
        }
        // Ends html code and closes file
        file.write("</table>\n</body>\n</html>");
        file.close();
        // Creates reader object to read file
        BufferedReader reader = new BufferedReader(new FileReader("numbers.html"));
        String line;
        while ((line = reader.readLine()) != null) {
            System.out.println(line);
        }
        reader.close();
    }
}
